### PR TITLE
remove unnecessary file level not unimplemented hide annotation

### PR DESCRIPTION
**Pull request by wbpcode**
Commit Message: remove unnecessary file level 'not unimplemented hide' annotation
Additional Description:

Remove the unnecessary file level 'not-implemented-hide' annotation. I found these annotations during the development of PR #18923 . But it seems that they should not have this annotation.

If this is a misunderstanding on my part, please close this PR.

Risk Level: Doc Only.
Testing: N/A.
Docs Changes: N/A.
Release Notes: N/A.

### CODE REVIEW

**Review by phlax — DISMISSED** [review]
afaict, removing these makes no difference - altho i guess with #18923 it will make a difference

https://storage.googleapis.com/envoy-pr/3b43372/docs/api-v3/service/discovery/v3/ads.proto.html

https://www.envoyproxy.io/docs/envoy/latest/api-v3/service/discovery/v3/ads.proto.html

so, lgtm

cc @htuch for final approval

File: api/envoy/service/endpoint/v3/leds.proto [hunk 1]
```diff
@@ -13,7 +13,6 @@ option java_multiple_files = true;
 option java_generic_services = true;
 option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
-// [#not-implemented-hide:]
```
[the remark this answers is not in the document]
Comment (wbpcode) [reply api/envoy/service/endpoint/v3/leds.proto:16]: get it.

**Comment by wbpcode** [comment]
cc @adisuissa @htuch

**Review by adisuissa — APPROVED** [review]
LGTM.
Leaving for @htuch to make sure I'm not missing anything.

**Review by htuch — APPROVED** [review]
LGTM, thanks!

**Pull request merged**